tests/e2e/file_volume_statefulsets.go

```diff
@@ -508,4 +508,186 @@ var _ = ginkgo.Describe("[csi-file-vanilla] File Volume statefulset", func() {
 		gomega.Expect(len(ssPodsAfterScaleDown.Items) == int(replicas)).To(gomega.BeTrue(),
 			"Number of Pods in the statefulset should match with number of replicas")
 	})
+
+	/*
+	   Test performs following operations
+
+	   Steps
+	   1. Create a storage class.
+	   2. Create nginx service.
+	   3. Create nginx statefulsets with 3 replicas.
+	   4. Wait until all Pods are ready and PVCs are bounded with PV.
+	   5. Restart CSI dameonset pods
+	   6. Scale down statefulsets to 2 replicas.
+	   7. Scale up statefulsets to 3 replicas.
+	   8. Scale down statefulsets to 0 replicas and delete all pods.
```

Comment:
When we scale down the sts, the pods should be auto-deleted right, why do we say "delete all pods" ?

Reply:
fixed the comment